### PR TITLE
Fix assessor availability tdg endpoint

### PR DESCRIPTION
**Pull request by mgilgar**
The create assessor user endpoint in Test Data Generator could not create the availability which has been fixed in this PR.

### CODE REVIEW

File: app/controllers/testdata/TestDataGeneratorController.scala [hunk 1]
```diff
@@ -146,8 +146,7 @@ trait TestDataGeneratorController extends BaseController {
       role = Some("assessor"),
       phone = Some("123456789"),
       assessor = Some(AssessorRequest(
-        skills = Some(List("ASSESSOR", "QUALITY_ASSURANCE_COORDINATOR")),
-        sifterSchemes = Some(List(SchemeId("GovernmentEconomicsService"), SchemeId("ProjectDelivery"), SchemeId("Sdip"))),
```
Comment (henricook) [inline app/controllers/testdata/TestDataGeneratorController.scala]: This looks like an accidental regression, is it?

Comment (mgilgar) [inline app/controllers/testdata/TestDataGeneratorController.scala]: Yes, it is. It is weird.

Comment (henricook) [inline app/controllers/testdata/TestDataGeneratorController.scala]: Just a merge that resolved wrong? Do you want to correct?

File: app/controllers/testdata/TestDataGeneratorController.scala [hunk 2]
```diff
@@ -165,7 +164,7 @@ trait TestDataGeneratorController extends BaseController {
       eventType = Some(EventType.FSAC),
       description = Some("PDFS FSB"),
       location = Some("London"),
-      venue = Some("LONDON_FSAC"),
+      venue = Some("London venue 1"),
```
Comment (henricook) [inline app/controllers/testdata/TestDataGeneratorController.scala]: This line probably also a merge issue?

Comment (mgilgar) [inline app/controllers/testdata/TestDataGeneratorController.scala]: Yes. It should be fixed now

File: app/controllers/testdata/TestDataGeneratorController.scala (hunk 1 above)
Comment (mgilgar) [inline app/controllers/testdata/TestDataGeneratorController.scala]: Yes, it should be fixed now.

**Pull request merged**